index.d.ts

```diff
@@ -0,0 +1,104 @@
+interface PromiseModule {
+	// This piece of code is obtained from https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/pify/index.d.ts.
```

Comment:
Instead of linking to where you got it from, the comment should explain what it does.